### PR TITLE
Clarified capabilities

### PR DESCRIPTION
**Pull request by aaronrothschild**
- added marketplace installation instructions
- Added more descriptive text
- Clarified usage section

### CODE REVIEW

**Comment by aaronrothschild** [comment]
@hanzei and @justinegeffen - I've updated the docs for the profanity filter.

File: README.md [hunk 1]
```diff
@@ -6,18 +6,22 @@
...
-This plugin allows you to censor profanity on your Mattermost server.
+This plugin allows you to censor profanity on your Mattermost server.  The plugin will check all messages before they are posted to any channel for matches against the "Bad Words List" that is configured.  Any word matches that a user tries to submit as a message, will be replaced with a series of "*" to replace the characters of that word.    
 
 **Supported Mattermost Server Versions: 5.2+**
 
-## Installation
+## Plugin Marketplace 
+
+1. Go to Main Menu > Plugin Marketplace in Mattermost
+2. Search for "Jira" or manually find the plugin from the list and click Install
```
Comment (nathanaelhoun) [inline README.md:16]:
```suggestion
2. Search for "Profanity Filter" or manually find the plugin from the list and click Install
```

File: README.md [hunk 2]
```diff
@@ -6,18 +6,22 @@
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-profanity-filter/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-profanity-filter/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
 
-This plugin allows you to censor profanity on your Mattermost server.
+This plugin allows you to censor profanity on your Mattermost server.  The plugin will check all messages before they are posted to any channel for matches against the "Bad Words List" that is configured.  Any word matches that a user tries to submit as a message, will be replaced with a series of "*" to replace the characters of that word.
```
Comment (justinegeffen) [inline README.md:9]:
```suggestion
This plugin allows you to censor profanity on your Mattermost server. The plugin will check all messages before they are posted to any channel for matches against the configured "Bad Words List". Any word matches that a user tries to submit as a message, will be replaced with a series of "*"s to replace the characters of that word.   
```

File: README.md [hunk 3]
```diff
@@ -6,18 +6,22 @@
...
 
-This plugin allows you to censor profanity on your Mattermost server.
+This plugin allows you to censor profanity on your Mattermost server.  The plugin will check all messages before they are posted to any channel for matches against the "Bad Words List" that is configured.  Any word matches that a user tries to submit as a message, will be replaced with a series of "*" to replace the characters of that word.    
 
 **Supported Mattermost Server Versions: 5.2+**
 
-## Installation
+## Plugin Marketplace 
+
+1. Go to Main Menu > Plugin Marketplace in Mattermost
```
Comment (justinegeffen) [inline README.md:15]:
```suggestion
1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
```

File: README.md (hunk 1 above)
Comment (justinegeffen) [inline README.md:16]:
```suggestion
2. Search for "Jira" or manually find the plugin from the list and click **Install**.
```

File: README.md [hunk 4]
```diff
@@ -6,18 +6,22 @@
...
+2. Search for "Jira" or manually find the plugin from the list and click Install
+3. After the plugin has downloaded and been installed, click the Configure button
```
Comment (justinegeffen) [inline README.md:17]:
```suggestion
3. Once the plugin has downloaded and been installed, click **Configure**.
```

File: README.md [hunk 5]
```diff
@@ -6,18 +6,22 @@
...
-This plugin allows you to censor profanity on your Mattermost server.
+This plugin allows you to censor profanity on your Mattermost server. The plugin will check all messages before they are posted to any channel for matches against the configured "Bad Words List". Any word matches that a user tries to submit as a message, will be replaced with a series of "*"s to replace the characters of that word.   
 
 **Supported Mattermost Server Versions: 5.2+**
 
-## Installation
+## Plugin Marketplace 
+
+1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
+2. Search for "Profanity Filter" or manually find the plugin from the list and click Install
```
Comment (hanzei) [inline README.md:16]:
```suggestion
2. Search for "Profanity Filter" or manually find the plugin from the list and click **Install**
```

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the docs update! One small request

**Comment by aaronrothschild** [comment]
@hanzei Looks good, thanks!

**Review by hanzei — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -6,18 +6,22 @@
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-profanity-filter/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-profanity-filter/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
 
-This plugin allows you to censor profanity on your Mattermost server.
+This plugin allows you to censor profanity on your Mattermost server. The plugin will check all messages before they are posted to any channel for matches against the configured "Bad Words List". Any word matches that a user tries to submit as a message, will be replaced with a series of "*"s to replace the characters of that word.
```
Comment (justinegeffen) [inline README.md:9]:
```suggestion
This plugin allows you to censor profanity on your Mattermost server. The plugin checks all messages for matches against the configured "Bad Words List" before they are posted to any channel. The characters in any word matches are replaced with a series of "*"s.
```

File: README.md [hunk 7]
```diff
@@ -6,18 +6,22 @@
...
-This plugin allows you to censor profanity on your Mattermost server.
+This plugin allows you to censor profanity on your Mattermost server. The plugin will check all messages before they are posted to any channel for matches against the configured "Bad Words List". Any word matches that a user tries to submit as a message, will be replaced with a series of "*"s to replace the characters of that word.   
 
 **Supported Mattermost Server Versions: 5.2+**
 
-## Installation
+## Plugin Marketplace 
+
+1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
+2. Search for "Profanity Filter" or manually find the plugin from the list and click **Install**
```
Comment (justinegeffen) [inline README.md:16]:
```suggestion
2. Search for "Profanity Filter" or find the plugin in the list and click **Install**.
```

File: README.md [hunk 8]
```diff
@@ -6,18 +6,22 @@
...
 
 1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-profanity-filter/releases) and download the latest release for your Mattermost server.
 2. Upload this file in the Mattermost System Console under **System Console > Plugins > Management** to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
 3. Activate the plugin at **System Console > Plugins > Management**.
 
 ### Usage
 
-Inside the `/server` directory, you will find the Go files that make up the server-side of the plugin. Within there, build the plugin like you would any other Go application.
-
 To configure what words are censored on your server, edit the bad words list in **System Console > Plugins > Profanity Filter > Bad words list**.
```
Comment (justinegeffen) [inline README.md:27]:
```suggestion
You can edit the bad words list in **System Console > Plugins > Profanity Filter > Bad Words list**.
```

**Comment by justinegeffen** [comment]
Non-blocker: @aaronrothschild, could you confirm whether it's "Bad Words List" or "Bad words list"? I can't verify in my test instance. Thank you!

Otherwise LGTM!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**